sources/updater/updog/src/bin/updata.rs

```diff
@@ -118,7 +118,7 @@ impl RemoveUpdateArgs {
         });
         // Note: We don't revert the maximum version on removal
         update_metadata::write_file(&self.file, &manifest)?;
-        if let Some(current) = manifest.updates.first() {
+        if let Some(current) = manifest.updates.last() {
```

Comment:
Is this just not sorted? I would expect to need to sort it to always get the max version?

Reply:
It is sorted in a reversed order. I will adjust the code to enforce a sort here so that we get deterministic result.